frontend/device/pocketbook/powerd.lua

```diff
@@ -10,27 +10,24 @@ void SetFrontlightState(int flstate);
 
 local PocketBookPowerD = BasePowerD:new{
     is_charging = nil,
+    fl_min = -1,
```

Comment:
Better use fl_min = 0 (some parts of the code assume 0 = frontlight off) - you can still, like you previously did, give -1 to hardware when you see 0.